### PR TITLE
Add reverse-string exercise.

### PR DESCRIPTION
**Pull request by vmichalak**

### CODE REVIEW

File: exercises/reverse-string/.meta/src/reference/kotlin/ReverseString.kt [hunk 1]
```diff
@@ -0,0 +1 @@
+fun reverse(input: String): String = input.reversed()
```
Comment (sjwarner-bp) [inline exercises/reverse-string/.meta/src/reference/kotlin/ReverseString.kt]: Some maintainers dislike the use of built-in functions, though I personally have little problem with it seeing as learning library functions is an important part of learning a language IMO! 🙂

File: config.json [hunk 2]
```diff
@@ -16,6 +16,16 @@
       ],
       "auto_approve": true
     },
+    {
+      "slug": "reverse-string",
+      "uuid": "0000e75e-9b43-11e8-98d0-529269fb1459",
+      "core": false,
+      "unlocked_by": null,
```
Comment (sjwarner-bp) [inline config.json]: Is this deliberately a 'bonus exercise' or is it worth specifying an `unlocked_by`?

File: exercises/reverse-string/.meta/src/reference/kotlin/ReverseString.kt (hunk 1 above)
Comment (vmichalak) [reply exercises/reverse-string/.meta/src/reference/kotlin/ReverseString.kt]: I changed the implementation in a new commit.

File: config.json (hunk 2 above)
Comment (vmichalak) [reply config.json]: I think it's better to make it a bonus exercise no ?

**Comment by vmichalak** [comment]
Wow, Sorry i forget this PR. It's now ok normally :)

**Comment by mikegehard** [comment]
Thank you!

I squashed the commits down into one here: https://github.com/exercism/kotlin/commit/7b3c307ba4e4c049db58cf40a32c71603110d282

**Comment by vmichalak** [comment]
Thanks @mikegehard

**Pull request closed without merging**